### ISSUE TITLE
pep8 unkown error

### ISSUE THREAD

**Issue by fffairforce** [fffairforce/bme547ci#3]
went through all the recommended amendment

**Comment by dward2** [comment]
Sorry, but I don't see a reference to any code or error.  Can you tell me exactly what error you are getting (cut and paste the error or a screen shot) and on what code?

**Comment by fffairforce** [comment]
![image](https://user-images.githubusercontent.com/42806161/52442939-6e26ce80-2af2-11e9-9b72-1abcee47a1ec.png)

**Comment by dward2** [comment]
Those all look like typical PEP-8 non-conformance errors.  So, you need to reduce line length by splitting line up, as you did in your original e-mail.  Also, you are getting the error that you only have one blank line between functions.  Add another line where indicated.

Error E225 is saying that “r=function” is not correct and should be “r = function”.

The other errors indicate what the problems are.  Are there any that you would like more explanation about?

**Comment by fffairforce** [comment]
I did the space and blank, But still get a error report. Since I add test.py with other interrupted error. I am now waiting. I re-upload the single file of tachycardic

**Comment by fffairforce** [comment]
Can't I ignore the other files when push? I did only add the specific a file on branch. But travis automatically  scanned the whole repository...

**Comment by dward2** [comment]
The default is that pytest will run everything.  You can edit the test files and "decorate" tests you don't want to run with `@pytest.mark.skip`, but then you will need to remove those later.

**Comment by fffairforce** [comment]
I see
![image](https://user-images.githubusercontent.com/42806161/52444094-66b4f480-2af5-11e9-8e55-3de6fb48e7eb.png)
Its still complaining about space, should I leave no space on left when drawing the half part to next line?

**Comment by dward2** [comment]
FYI, if you want more info on customizing pytest, visit https://docs.pytest.org/en/latest/example/pythoncollection.html#customizing-test-collection

**Comment by fffairforce** [comment]
Ok, I will look up for it. Thanks!

**Comment by dward2** [comment]
The first error above is that there are four spaces on line 15.  PEP-8 doesn't like spaces on empty lines.

The second error above is the fact that there is a space after the last character.  Again, PEP8 doesn't like this.  It tries to avoid spaces that aren't necessary.

The third error is about indentation for readability.  it probably wants you to add more spaces so that the code looks like this with the quote of the continuation line being where the `(` is on the line above.  Or, it may need another space.  Keep changing indents until the error goes away.:
```
r = difflib.SequenceMatcher(None, str(instring).lower(),
                           "tachycardic").ratio()
```

The fourth error is the same as the first error.

**Comment by dward2** [comment]
Actually, for the third error, look in your file and compare line 18 with line 23.  You didn't get the indentation error for line 18, but did for line 23.  So, change line 23 to be like line 18.

**Comment by fffairforce** [comment]
well, we shall need a space after punctuation, I will try it out.
Does the third one mean I will have to test and wait 5min every time upload a single space adjustment? I might better use conda to test locally first right ....

**Comment by dward2** [comment]
Yes, I would mostly run pytest locally.  Travis is really intended as last check testing before a merge.  Ongoing changes should be checked locally.
